MANUAL_REGRESSION.md

```diff
@@ -449,20 +449,22 @@ Given there is no webcam connected to the computer
 Given user opened the link with `?uploadFallback=false` flag
 
 1. And user is on first page of cross-device flow
-2. Open the cross device link on mobile browsers with a working webcam.
+2. Open the cross device link on mobile browsers with a working camera.
     - user should be able to complete the cross-device flow successfully.
 
 Given user opened the link with `?uploadFallback=false` flag
 
 1. And user is on first page of cross-device flow
-2. Open the cross device link on mobile browsers with a malfunctioning webcam or on mobile browsers that do not support getUserMedia (ie Safari on iOS10.3 or earlier).
+2. Open the cross device link on mobile browsers with a malfunctioning camera or on mobile browsers that do not support getUserMedia (i.e. Safari on iOS10.3 or earlier, Chrome on iOS)
     - user won't see the "use the native camera mode instead" link
+    - user should see `Unsupported browser` message
+    - user should see `Restart the process on Safari` message
```

Comment:
I think we should specify which browser user should see which message.

`user should see _Restart the process on Safari_ message if on Google Chrome`

`user should see _Restart the process on Google Chrome_ message if on Safari`

Would that make sense?

Reply:
Good idea. I think it would cover most cases. There will/might be edge cases like a user with Safari on iOS10 (for example) or using Firefox mobile.

Reply:
I think that doing browser sniffing is never a good idea. For users that are on older versions of iOS this error message might not provide any help and might be confusing. Wouldn't it be better to provide a list of valid alternatives for the user to restart the flow on that is not based on the current browser they are on?

Reply:
I did raise the point about the wording being confusing for users on older iOS versions (same would apply for Android Chrome I imagine). Though I think in terms of getUserMedia support the user is limited to Safari on iOS11+ and Google Chrome on (newer?) Androids?

Reply:
Yes, in that case it might be worth adding the version and have the same message for both iOS and Android so we don't need to do browser sniffing because it's not reliable

Reply:
Will bring this up with team for discussion to confirm the messaging we want to show users.

Reply:
After discussion, changed messaging to the following:
`Restart the process on the latest version of Google Chrome` (Android)
`Restart the process on the latest version of Safari` (iOS)